src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -194,6 +194,7 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, policyMembers, r
             icon: Expensicons.Tax,
             action: singleExecution(waitForNavigate(() => Navigation.navigate(ROUTES.WORKSPACE_TAXES.getRoute(policyID)))),
             routeName: SCREENS.WORKSPACE.TAXES,
+            brickRoadIndicator: PolicyUtils.hasTaxRateError(policy) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
```

Comment:
NAB, It seems this change is missed from [the docs](https://docs.google.com/document/d/12FZycslrIE95QoKM8DZ37H_wLbH9w1jt3uA-a6BCqtc/edit#heading=h.zeqehpn6omdq). 

How can I make an error?